merqube_client_lib/templates/equity_baskets/creators.py

```diff
@@ -0,0 +1,64 @@
+"""
+equity basket creators
+"""
+import logging
+from typing import Any
+
+import pandas as pd
+
+from merqube_client_lib.logging import get_module_logger
+from merqube_client_lib.pydantic_types import (
+    ClientMultiEBConfig,
+    ClientTemplateResponse,
+)
+from merqube_client_lib.templates.equity_baskets.schema import (
+    ClientDecrementConfig,
+    ClientMultiEquityBasketConfig,
+    ClientSSTRConfig,
+)
+from merqube_client_lib.templates.equity_baskets.util import EquityBasketIndexCreator
+
+logger = get_module_logger(__name__, level=logging.DEBUG)
+
+
+def read_file(filename: str) -> list[Any]:
+    """reads portfolio/overrides files"""
+    return list(pd.read_csv(filename, float_precision="round_trip").to_dict(orient="index").values())
```

Comment:
This utility its going to read just file from us? If its the case, you can ignore this comment